po/de.po

```diff
@@ -682,10 +682,10 @@ msgid ""
 "use. Click the triangle at the far right of the column headings to change "
 "the display."
 msgstr ""
-"Sie können die Wertpapier-Editor Anzeige die «Quelle» zu einem Wertpapier "
+"Sie können sich im Wertpapier-Editor die «Quelle» zu einem Wertpapier "
 "anzeigen lassen und so die Herkunft der online-Daten sehen. Klicken Sie dazu "
```

Comment:
`Online-Daten`

Reply:
@fellen: habe ich in Weblate korrigiert.